ts/types/ember-cli-blueprint-test-helpers/index.d.ts

```diff
@@ -1,6 +1,8 @@
 declare namespace Chai {
   interface ChaiStatic {
-    file(name: string): {
+    file(
+      name: string
+    ): {
```

Comment:
A little surprised by this one 🤔

Reply:
Yeah, that's odd.

Reply:
I'm surprised by this one also. Heh.

Reply:
Looks like, weirdly, Prettier prefers to wrap the params across multiple lines if the return type is wrapped across multiple lines. Quirky!